gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #140: Overall Beautification

### Description

I redesigned ShowUp to hopefully look more beautiful.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
```

1. Get the code from this PR
1. Run `pip install django-crispy-forms django-filter`
1. Run the Django server and go to the local server URL
1. See the new homepage
> ![image](https://user-images.githubusercontent.com/13837978/68551281-40e8b300-03d9-11ea-9c29-e3f01c1547a7.png)
5. Click on "Sign up" and see the new signup page and fill it out
> ![image](https://user-images.githubusercontent.com/13837978/68551293-5fe74500-03d9-11ea-95ca-55e2f88c0483.png)
6. Get the email, confirm it, and sign in. See the new login page.
> ![image](https://user-images.githubusercontent.com/13837978/68551304-8b6a2f80-03d9-11ea-816f-1d3b17f68484.png)
7. See the new home page
> ![image](https://user-images.githubusercontent.com/13837978/68551315-a341b380-03d9-11ea-9264-7675cd961f1c.png)
8. Go to the events page and see the new design. See that when you scroll down, the filters remain visible. See that for events that you're interested in or going to, the name of the event is now a link to the matching stack for that event.
> ![image](https://user-images.githubusercontent.com/13837978/68551323-bf455500-03d9-11ea-93fa-d5f2a2e80412.png)
9. Mark some events as interested and some as going. Mark other users as interested and going to the same events. Go to the homepage and go to the matching stack for an event and see the new design. Test the swiping.
10. Go to your user profile and see the new design.

### Miscellaneous Comments

* I changed pretty much everything so test very thoroughly. I can't write out every single test case so be creative.
* I used the Bootstrap framework. One of its features is that it's responsive to different screen resolutions. Some modern laptops have very high resolutions but mine doesn't so I can't test that. @jawooson and maybe @mksavic, I think yours do so please pay careful attention to the layout of the pages. It all looks good for me but it might look weird for you so please let me know.
* Thanks to @Vedanth15 for help with the signup and matching stack pages.
* @mksavic when you've done issue #139, please change the "Matches" link in the header to point to your new page. 

### Checklist
